addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/generators/BodyGenerator.java

```diff
@@ -122,6 +129,22 @@ private String generateFromArraySchema(ArraySchema schema) {
         return createJsonArrayWith(generate(schema.getItems()));
     }
 
+    private String generateFromBinarySchema(BinarySchema schema, Boolean isImage) {
```

Comment:
- `static`
 - `Boolean` → `boolean`

Reply:
@thc202 Making this method static would require making the public strings `imageFileContents` and `textFileContents` static too. Then they cannot be referenced in the tests (as `generators.getBodyGenerator()` returns an object of the `BodyGenerator` class, and trying to access the strings gives the error `warning: [static] static variable should be qualified by type name, BodyGenerator, instead of by an expression`. So should I import the entire `BodyGenerator` class into the tests seperately so as to access the strings via `BodyGenerator.textFileContents`? Or should this method be left non-static?

Reply:
Import `BodyGenerator` as needed (note that constants should be all uppercase and words separated with underscores).